### PR TITLE
TOOLS/UCT/TCP: Print cm info in ucx_info and use cm config in tcp_sockcm

### PR DESCRIPTION
**Pull request by alinask**

### CODE REVIEW

File: src/tools/info/tl_info.c [hunk 1]
```diff
@@ -471,6 +471,66 @@ static void print_md_info(uct_component_h component,
...
+static void print_cm_info(uct_component_h component,
+                          const uct_component_attr_t *component_attr)
+{
+    ucs_async_context_t *async;
+    uct_worker_h worker;
+    ucs_status_t status;
+
+    status = ucs_async_context_create(UCS_ASYNC_MODE_THREAD_SPINLOCK, &async);
+    if (status != UCS_OK) {
+        return;
```
Comment (dmitrygx) [inline src/tools/info/tl_info.c]: should we add `printf("# < failed to create asynchronous context>\n");`?

File: src/tools/info/tl_info.c [hunk 2]
```diff
@@ -471,6 +471,66 @@ static void print_md_info(uct_component_h component,
...
+    }
+
+    status = uct_worker_create(async, UCS_THREAD_MODE_SINGLE, &worker);
+    if (status != UCS_OK) {
+        goto out_async_ctx_destroy;
```
Comment (dmitrygx) [inline src/tools/info/tl_info.c]: should we add `printf("# < failed to create worker>\n");`?

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>PASSED</b#  > on 25 workers (click for details)</summary>

**Note:** the logs will be deleted after 18-Dec-2019

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/6e42453dd77b376d9d1cb37efbf8d481/raw/e565bb72299ecf4def825d698c3e913ece26b11f/_main) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/6e42453dd77b376d9d1cb37efbf8d481/raw/efa2ab5125de1ae00e136ed385d9813be4700e62/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/6e42453dd77b376d9d1cb37efbf8d481/raw/da9be9ecdf3d2f5828098df01feb32afc80d31ff/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/6e42453dd77b376d9d1cb37efbf8d481/raw/76f1b8b71454c3cfd4ae21cb397b6cb54555880b/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/6e42453dd77b376d9d1cb37efbf8d481/raw/05a6cab6e5752a51eb5a024d6aa09d115d6f33c8/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/6e42453dd77b376d9d1cb37efbf8d481/raw/507b3b0d81ac76f4698f5ca025b1c188ee0a6fc4/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/6e42453dd77b376d9d1cb37efbf8d481/raw/b3be4949c68bba764359b49cdaad82f2e4a55de3/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/6e42453dd77b376d9d1cb37efbf8d481/raw/397d08c2170c4a32482165511cc9428959c80f98/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/6e42453dd77b376d9d1cb37efbf8d481/raw/8e8879b83349e1ddd728275c8a03b7a2b27192fe/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/6e42453dd77b376d9d1cb37efbf8d481/raw/1e62fba9bffb83fbd65dc7839089c1772a8832d7/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/6e42453dd77b376d9d1cb37efbf8d481/raw/b1f73989cfb34f16d256242af31dafcb7372a34d/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/6e42453dd77b376d9d1cb37efbf8d481/raw/deb239ceaf58e0a182c115db8c7d6653f414bed4/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/6e42453dd77b376d9d1cb37efbf8d481/raw/75a7fc8307956d9bd3c6c4a28131e1218bf6cad8/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/6e42453dd77b376d9d1cb37efbf8d481/raw/53200011ea51d159ad9db917e27227f4d9e88dcf/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/6e42453dd77b376d9d1cb37efbf8d481/raw/46c99cd765e6e39853bfb0be26fe0cb039312a86/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/6e42453dd77b376d9d1cb37efbf8d481/raw/e948de20f0816f232deb6aa68e6bb4c71c0e9196/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/6e42453dd77b376d9d1cb37efbf8d481/raw/811252a75dad9aa760acd1a8156075d2ec96b47e/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/6e42453dd77b376d9d1cb37efbf8d481/raw/3b5a8f097ac860fccc84e3734d35ff5a200bf695/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/6e42453dd77b376d9d1cb37efbf8d481/raw/f95f0a86bcb2247915232baeb022d4ea29201ecb/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/6e42453dd77b376d9d1cb37efbf8d481/raw/1d6ca5cd5a4a5b2f8b8cc2d1e07d0cc341493022/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/6e42453dd77b376d9d1cb37efbf8d481/raw/0088df5da66318fae573e9c840fd9a59b4d02695/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/6e42453dd77b376d9d1cb37efbf8d481/raw/d80437e9a5bef56f3645ec94b71326df64f7313e/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/6e42453dd77b376d9d1cb37efbf8d481/raw/217c32415fa05d293f2a7ea2a57c9eeb496e95aa/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/6e42453dd77b376d9d1cb37efbf8d481/raw/0ad401c045b9d1ea852544ce6b6e53f76b277ff0/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/6e42453dd77b376d9d1cb37efbf8d481/raw/b9b21967554456366b9d9427996a2ca85325d19e/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>PASSED</b#  > on 25 workers (click for details)</summary>

**Note:** the logs will be deleted after 18-Dec-2019

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/48cb1a46854814c103f2648aa006ad96/raw/45ae3c0251882a4b4442cfb78e6eb16710b33dc9/_main) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/48cb1a46854814c103f2648aa006ad96/raw/f4c14c697a71ad7e926a31e333ca3cc86ade898c/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/48cb1a46854814c103f2648aa006ad96/raw/18d9c17aeb4868c8a4f00b03dd7080936778f2f8/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/48cb1a46854814c103f2648aa006ad96/raw/b32dcba06c67b469c07105ed5104a910314d9d7a/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/48cb1a46854814c103f2648aa006ad96/raw/76a45ce3a387d23a8ee0d5dd864ff23ceb801d61/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/48cb1a46854814c103f2648aa006ad96/raw/a1d592f8c1eea6bf96c9eb0c8ad11c82de352948/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/48cb1a46854814c103f2648aa006ad96/raw/952343a9e57390847a4598320cd4802945087664/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/48cb1a46854814c103f2648aa006ad96/raw/2f04712436b2364cd48e42b7f698f87637899449/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/48cb1a46854814c103f2648aa006ad96/raw/4372dee1eeb27014a0b61284230c736188400b8e/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/48cb1a46854814c103f2648aa006ad96/raw/10c12c7a7775a2ef1f51696190f283227ade7e01/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/48cb1a46854814c103f2648aa006ad96/raw/22853699d0e1f311edc21e3179994103ed68373f/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/48cb1a46854814c103f2648aa006ad96/raw/695130009c38d9abddaab0179c1a9a62bce66b3e/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/48cb1a46854814c103f2648aa006ad96/raw/0a9b4075c65381bac4908d95b7db311317a8eeda/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/48cb1a46854814c103f2648aa006ad96/raw/e39b84766975bcf58962b541ccb1a8fc665969f8/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/48cb1a46854814c103f2648aa006ad96/raw/4776bc25c580bdd89ae617ccffe0fd94633bd1b6/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/48cb1a46854814c103f2648aa006ad96/raw/2b62e4b865c85c42d58e90a88e15030003c6ecb3/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/48cb1a46854814c103f2648aa006ad96/raw/447fbc05f08e45318404c6c380e3cc2f74d5301d/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/48cb1a46854814c103f2648aa006ad96/raw/92c9e6a59febab94d7830e7152176bbb06697b97/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/48cb1a46854814c103f2648aa006ad96/raw/bc523730eca8dd4d277ebda44828f81c80d69e91/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/48cb1a46854814c103f2648aa006ad96/raw/612904b855b0365062fd963a0cf03fccbdc3d459/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/48cb1a46854814c103f2648aa006ad96/raw/378886543a4ef016a346b4e116b729e83076a75e/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/48cb1a46854814c103f2648aa006ad96/raw/5cad6aad259b69353925c416cfd7429d569eb62d/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/48cb1a46854814c103f2648aa006ad96/raw/5f53f0249775be39fc4456253f0bd46e067079cd/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/48cb1a46854814c103f2648aa006ad96/raw/220481393901e603ddd4b32c1f9c857998949f80/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/48cb1a46854814c103f2648aa006ad96/raw/0be6bfd53debde118f8e36b4e3634a95593ec3d0/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

File: src/tools/info/tl_info.c [hunk 3]
```diff
@@ -471,6 +471,70 @@ static void print_md_info(uct_component_h component,
...
+    }
+
+    cm_attr.field_mask = UCT_CM_ATTR_FIELD_MAX_CONN_PRIV;
+    status = uct_cm_query(cm, &cm_attr);
+    if (status != UCS_OK) {
+        printf("# < failed to query connection manager >\n");
+    } else {
+        printf("#\n");
+        printf("# Connection manager: %s\n", comp_name);
+        printf("#     Component: %s\n", comp_name);
```
Comment (yosefe) [inline src/tools/info/tl_info.c]: the `:` should be on same column when it's printed

File: src/tools/info/tl_info.c [hunk 4]
```diff
@@ -471,6 +471,70 @@ static void print_md_info(uct_component_h component,
...
+        return;
+    }
+
+    cm_attr.field_mask = UCT_CM_ATTR_FIELD_MAX_CONN_PRIV;
+    status = uct_cm_query(cm, &cm_attr);
+    if (status != UCS_OK) {
+        printf("# < failed to query connection manager >\n");
+    } else {
+        printf("#\n");
+        printf("# Connection manager: %s\n", comp_name);
```
Comment (yosefe) [inline src/tools/info/tl_info.c]: why need to print comp_name twice?

**Comment by alinask** [comment]
storage issues.

bot:pipe:retest

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>PASSED</b#  > on 25 workers (click for details)</summary>

**Note:** the logs will be deleted after 22-Dec-2019

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/865ebb01c99d23acce81bab332851378/raw/d8b22fc40ea1bd4730c3e1c70d4176dde43e6e30/_main) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/865ebb01c99d23acce81bab332851378/raw/1a1cc9d69689a32156fe8b5e808a8db279a5baef/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/865ebb01c99d23acce81bab332851378/raw/f87ad4fda3ddc760daf97c0c3b5ebda650dc328e/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/865ebb01c99d23acce81bab332851378/raw/aebae071b86ed62588d79eab434d43471066d90d/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/865ebb01c99d23acce81bab332851378/raw/80177a3b061ca6944262118cafc4c4242e9700c7/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/865ebb01c99d23acce81bab332851378/raw/6b8b294bebf323a027cd71536c2b8fe0d97e3b94/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/865ebb01c99d23acce81bab332851378/raw/91ddb46f30e6562a775431c7ce1da770196c2d72/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/865ebb01c99d23acce81bab332851378/raw/c266cbc17f0f36cfb4731edf8192bcb8ebcc27e0/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/865ebb01c99d23acce81bab332851378/raw/3482f2fee79deb39d5d38403359c4f38380c1e7e/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/865ebb01c99d23acce81bab332851378/raw/db888cb6b44b30960b7e03b25e9e6ef368ce545e/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/865ebb01c99d23acce81bab332851378/raw/d524c721e189495be1b2e4523acc004b3dc91735/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/865ebb01c99d23acce81bab332851378/raw/04b58c39e7f4bc16b87c2686bd8fc606ee39b41f/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/865ebb01c99d23acce81bab332851378/raw/7d181b555e9efcb87363ac0406ce4026f697bd11/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/865ebb01c99d23acce81bab332851378/raw/19944df31d612be43572e2bdd0d462f7db710bb5/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/865ebb01c99d23acce81bab332851378/raw/c0c14d95a34fa71bebf321f3145b8e6ba9dbc43a/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/865ebb01c99d23acce81bab332851378/raw/5fd4b8e3e02b7f045a81f2107d47fcf0c3e02a0f/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/865ebb01c99d23acce81bab332851378/raw/402c4bad98bdefbd7eff98385882be851eb4e340/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/865ebb01c99d23acce81bab332851378/raw/bd15d84f7725121fc9da5b6d7dd3c4dedc1d7671/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/865ebb01c99d23acce81bab332851378/raw/bb1bfda3a1214543d4a0a061a9d09cfa9e2744c8/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/865ebb01c99d23acce81bab332851378/raw/19fc56c4185742a4e94e0bbe0ebca7650bcbb292/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/865ebb01c99d23acce81bab332851378/raw/12f775b0166169b7896f1e2554b8765bc5826642/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/865ebb01c99d23acce81bab332851378/raw/954ea9797147509a05a71ef33c3507c2b23cb455/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/865ebb01c99d23acce81bab332851378/raw/146f4f3b4865e824f7f71e24ad4c58d700b66798/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/865ebb01c99d23acce81bab332851378/raw/c057fb4b45bf39045f516b356529c2438bbcc388/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/865ebb01c99d23acce81bab332851378/raw/4ad83324f2b16f515646829a8fa9ecd95e3af2f1/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

**Comment by alinask** [comment]
@yosefe good to go?

**Pull request merged**